Sources/ProjectDescription/TargetAction.swift

```diff
@@ -70,14 +74,16 @@ public struct TargetAction: Codable, Equatable {
     ///   - outputPaths: List of output file paths.
     ///   - outputFileListPaths: List of output filelist paths.
     ///   - basedOnDependencyAnalysis: Whether to skip running this script in incremental builds
+    ///   - runOnlyForDeploymentPostprocessing: Whether this script only runs on install builds (default is false)
     init(name: String,
          script: Script = .embedded(""),
          order: Order,
          inputPaths: [Path] = [],
          inputFileListPaths: [Path] = [],
          outputPaths: [Path] = [],
          outputFileListPaths: [Path] = [],
-         basedOnDependencyAnalysis: Bool? = nil)
+         basedOnDependencyAnalysis: Bool? = nil,
+         runOnlyForDeploymentPostprocessing: Bool = false)
```

Comment:
We may want to follow a similar approach of using optional Bool here.

Reply:
was using the same approach as `PBXShellScriptBuildPhase` with the default value...

Reply:
feel free to change to `: Bool? = nil`...